highfidelity/hifi
# WL 21386: ShortBow Improvements

# Test Plan

1. Confirm that you can hear a sound effect and you can see a smoke effect appear for one second when you hit a ball with an arrow.
2. Confirm that the game ends okay and doesn't hang waiting for any balls.

Test this in your Local Sandbox, and running the 'startShortbow.js' script inside the zip attached. [shortbow.zip](https://github.com/highfidelity/hifi/files/1089823/shortbow.zip)

NOTE: Sound effect is a placeholder and will be swapped out prior to merging.

Per worklist https://worklist.net/21386
